uint16 lift_active
uint16 mode_up=1
uint16 mode_down=0